config/modules/all.conf

```diff
@@ -8,3 +8,6 @@ include subset/security/build.conf
 include subset/cloud/build.conf
 include subset/manual/build.conf
 include subset/network/build.conf
+
+# DHCP configuration.
```

Comment:
comment is redundant/unnecessary

What's the broader category this falls under?  EIther it should be under "baseline" if it's core to the system, or it should just be merged with host.conf, or we need to categorize it appropriately. 

My guess is that dhcp.conf should just be merged into host.conf -- but I see you explicitly pulled the ipaddr tests out from there... why?  Is it that there's expected to be more "dhcp tests" in the future?

Reply:
This is to ensure ipaddr and DHCP runs after all other tests. Otherwise the modified network configuration [interferes with other tests (notably security.passwords)](https://github.com/faucetsdn/daq/runs/970603327). An alternative approach would be to take a second look at how ipaddr and the faux device behaves after the functions have finished (ideally returning to the original IP address and configuration).

Reply:
Update comment to say why, not what.  https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/